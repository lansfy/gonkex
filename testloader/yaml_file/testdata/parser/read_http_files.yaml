- name: loader MUST read upload files requests
  method: POST
  headers:
    Content-Type: multipart/form-data
  path: /some/path
  form:
    files:
      file1: "testdata/upload-files/file1.txt"
      file2: "testdata/upload-files/file2.log"
  response:
    200: "aaabbb"
