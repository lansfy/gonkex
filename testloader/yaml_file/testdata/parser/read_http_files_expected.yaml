- GetName: loader MUST read upload files requests
  GetMethod: POST
  Path: /some/path
  ContentType: multipart/form-data
  Headers:
    Content-Type: multipart/form-data
  GetForm:
    GetFiles:
      file1: "testdata/upload-files/file1.txt"
      file2: "testdata/upload-files/file2.log"
  GetResponses:
    200: "aaabbb"
  GetFileName: testdata/parser/read_http_files.yaml
  FirstTestInFile: true
  LastTestInFile: true
